pmultiqc/modules/proteobench/proteobench_utils.py

```
@@ -426,10 +436,14 @@ def draw_logintensitymean_vs_logfc(df):
         "ylab": "logIntensityMean",
     }
 
-    scatter_html = scatter.plot(data=plot_data, pconfig=draw_config)
+    scatter_html = scatter.plot(
+        data=plot_data,
+        pconfig=draw_config
+    )
 
-    return scatter_html
+    scatter_html == remove_subtitle(scatter_html)
```

Ruff (0.11.9)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)